taskcat/amiupdater.py

```diff
@@ -0,0 +1,369 @@
+import collections
```

Comment:
pycharm picks up 6 warnings and 29 weak warnings for this file, can we get as many as possible addressed, alot are just PEP8, but some that look like they may manifest as bugs and are not in the tested codepath.